### DISCUSSION TITLE
Author Script to Analyze VBET Observation Results

### DISCUSSION THREAD

**Discussion by shelbysawyer** [Riverscapes/riverscapes-tools#336] [VBET] [answered]
I'm creating this discussion as a place to drop my draft script to get feedback.

I will author a script in R that will create boxplots, distributions, and summary statistics for the ~2800 points recorded in the VBET validation postGIS database.

**Comment by joewheaton** [comment]
Hey @shelbysawyer how's this going? You might ask @KellyMWhitehead if there is a place you should be putting your script in this or another repo and if this is an action verb ("author") do that as an issue. And reference it in your commits.

**Reply by shelbysawyer** [reply]
I'm struggling with it, but getting some help today. I can delete this and create an issue, or just leave this and create an issue from this. I'll wait to hear from @KellyMWhitehead if this the right place.

**Comment by joewheaton** [comment, marked as answer]
This is now outdated and has been done.
